src/assets/keycloak.json

```diff
@@ -0,0 +1,10 @@
+{
+  "realm": "ndb-dev",
```

Comment:
is there any danger of having this checked into git and probably built into the docker image by default. if someon forgets to provide a custom file during deployment? I guess worst case the instance will try to auth with our dev server, which is not a big issue?

Reply:
Yes, I think it is fine. The keycloak instance is public and secured. It is the same as we have the `dev.aam-digital.com` as the checked-in default database. We could think about having a dev and a production keycloak though.

Reply:
yes, in fact we were talking about splitting servers also. Let's plan for that and use a different tld (maybe .net) for testing and dev